src/main/java/com/meta/cp4m/llm/HuggingFaceLlamaPlugin.java

```diff
@@ -66,51 +72,6 @@ public T handle(ThreadState<T> messageStack) throws IOException {
         String llmResponse = allGeneratedText.strip().replace(prompt.strip(), "");
```

Comment:
the response has the prompt in it?

Reply:
Yeah Huggingface returns the entire block including the original prompt